pymc3/tests/test_data_container.py

```diff
@@ -156,6 +157,15 @@ def test_shared_data_as_rv_input(self):
         np.testing.assert_allclose(np.array([2.0, 4.0, 6.0]), x.get_value(), atol=1e-1)
         np.testing.assert_allclose(np.array([2.0, 4.0, 6.0]), trace["y"].mean(0), atol=1e-1)
 
+    def test_shared_variable_as_mu(self):
```

Comment:
In hindsight I think it is probably to wasteful to sample in this unittest, and it also is not addressing the most important part: that changes in the shared variable value are reflected in the model.

I think it is probably enough to check the logp. Something like this:
```python
with pm.Model() as m:
    shared_var = theano.shared(5.0)
    v = pm.Normal("v", mu=shared_var, shape=1)

np.testing.assert_allclose(
    v.logp({'v': 5.0}),
    -0.91893853,
    rtol=1e-5,
)

shared_var.set_value(10.0)

np.testing.assert_allclose(
    v.logp({'v': 10.0}),
    -0.91893853,
    rtol=1e-5,
)
```

Reply:
Also, let's give it a more specific name like `def test_shared_scalar_as_rv_input` ?